src/Features/Core/Portable/CodeRefactorings/OrderableLanguageDocumentMetadata.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings;
+
+internal sealed class OrderableLanguageDocumentMetadata : OrderableLanguageMetadata
```

Comment:
If this type is only being used in the Distribute... method, keep this private to the class?

Reply:
Good point